### ISSUE TITLE
Seminal vesicle agenesis                                                                                                                                                                                                                     Seminal vesicle agenesis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10326]
**Preferred term label:**

**Synonyms**
absent seminal vesicles

**Definition (free text, please give PubMed ID)**
PMID: 24839497 Seminal vesicle malformations are a rare cause of obstructive azoospermia, often associated with other internal genitalia and upper urinary tract birth defects. We report 5 new cases of **seminal vesicle agenesis** in men presenting with hypospermia and azoospermia. Imaging showed seminal vesicle unilateral agenesis in all patients.
The remaining seminal vesicle was hypoplastic in 3 cases, dilated in 1 case and with abnormally thick content in another case. Vas
deferens agenesis was observed unilaterally in 2 patients and bilaterally in 2 other patients.

PMID: 30136289 Cases of CBAVD account for 1%–2% of infertile men, 4%–17% of azoospermic men and up to 25% of those with obstructive azoospermia. The condition is typically accompanied by preservation of normal testicular exocrine and endocrine function and normal spermatogenesis (Singh, Hamada, Bukavina, & Agarwal, 2012). Furthermore, it is transmitted as an autosomal recessive disease and may be isolated or is associated with mutations of the cystic fibrosis transmembrane regulator protein, which lead to absent vasa as well as partial or complete absence of the epididymis. Isolated cases may manifest other congenital anomalies, such as **absent seminal vesicles** and/or unilateral kidney agenesis

**Parent term (use hpo.jax.org/app)**
Abnormal Seminal vesicle morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:48   Congenital bilateral absence of vas deferens

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f   added as HP:0430121
